src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Services/SyntaxFacts/ISyntaxFacts.cs

```diff
@@ -333,14 +333,14 @@ internal partial interface ISyntaxFacts
         /// In VB, this includes all block statements such as a MultiLineIfBlockSyntax.
         /// </summary>
         bool IsExecutableBlock(SyntaxNode node);
-        SyntaxList<SyntaxNode> GetExecutableBlockStatements(SyntaxNode node);
+        IReadOnlyList<SyntaxNode> GetExecutableBlockStatements(SyntaxNode node);
         SyntaxNode FindInnermostCommonExecutableBlock(IEnumerable<SyntaxNode> nodes);
 
         /// <summary>
         /// A node that can host a list of statements or a single statement. In addition to
         /// every "executable block", this also includes C# embedded statement owners.
         /// </summary>
-        bool IsStatementContainer(SyntaxNode node);
+        bool IsStatementContainer([NotNullWhen(true)] SyntaxNode node);
```

Comment:
Seems confusing to have `NotNullWhen` attribute in files which are not nullable enabled yet.

Reply:
Enabled NRT for this method